### ISSUE TITLE
Field level format, parse, and validate.

### ISSUE THREAD

**Issue by PaulSavignano** [final-form/react-final-form-hooks#6]
### **feature request**

Would like guidance on implementing field level validation, parse, and format functions.

**Comment by erikras** [comment]
Yes, `parse`/`format` is functionality from RFF that wasn't implemented here. I'll think about how best to add it.

The field-level validation could be added easily via the "field config" prop to `registerField()`.

Let me think on both of these... 🤔

**Comment by PaulSavignano** [comment]
Thanks for the update here Erik!  I'll check into the registerField() prop.

**Comment by erikras** [comment]
I didn't mean for you to close this. It's still a legit feature request.

**Comment by RIP21** [comment]
I didn't get what do you mean by "field config" prop to `registerField()` :) Can you show any code example maybe?

**Comment by baharev** [comment]
Seconded. Please provide some examples.
